hdr_schemata/utils/create_markdown.py

```diff
@@ -234,18 +237,18 @@ def remove_types(data):
 from hdr_schemata.models.GWDM.v2_0 import Gwdm20
 
   
-create_markdown(Hdruk220, "./docs/HDRUK/", "2.2.0")
-create_markdown(Hdruk221, "./docs/HDRUK/", "2.2.1")
-create_markdown(Hdruk212, "./docs/HDRUK/", "2.1.2")
-create_markdown(Hdruk213, "./docs/HDRUK/", "2.1.3")
-create_markdown(Hdruk300, "./docs/HDRUK/", "3.0.0")
+# create_markdown(Hdruk220, dir_path+"/../../docs/HDRUK", "2.2.0")
+# create_markdown(Hdruk221, dir_path+"/../../docs/HDRUK", "2.2.1")
+# create_markdown(Hdruk212, dir_path+"/../../docs/HDRUK", "2.1.2")
+# create_markdown(Hdruk213, dir_path+"/../../docs/HDRUK", "2.1.3")
```

Comment:
uncomment as we want to clean the markdown for some of the other old schemas too?

Reply:
I'll do that in the next PR if you're happy with that. Didn't want to load in too many changes all in one go 🤣